sdk/monitor/opentelemetry-exporter/samples/@azure/storage-blob/typescript/src/tracing.ts

```diff
@@ -0,0 +1,22 @@
+import { AzureMonitorTraceExporter } from "@azure/monitor-opentelemetry-exporter";
+import { NodeTracerProvider } from "@opentelemetry/node";
+import { BatchSpanProcessor } from "@opentelemetry/tracing";
+import * as azureSdkTracing from "@azure/core-tracing";
+
+const provider = new NodeTracerProvider();
+
+const azureExporter = new AzureMonitorTraceExporter({
+  instrumentationKey: "ikey",
+  logger: provider.logger,
+});
+
+provider.addSpanProcessor(new BatchSpanProcessor(azureExporter, {
+  bufferSize: 1000, // 1000 spans
+  bufferTimeout: 15000, // 15 seconds
+}));
+
+provider.register();
+
+azureSdkTracing.setTracer(
+  provider.getTracer("example tracer") as any // cast to any until azure sdk uses latest types
```

Comment:
nit: might be better to cast through unknown to get to the shape you want e.g. `as unknown as ExpectedShape`

Reply:
For now it complains about the `Tracer` defined by `@opentelemetry/types` which doesn't exist anymore, not exactly matching the one defined by `/api`.